### ISSUE TITLE
TypeError when sending multiple galleries to Library or Trash

### ISSUE THREAD

**Issue by nonamethanks** [happypandax/happypandax#180]
**Describe the issue**
Pretty much every single action in view actions fails with a TypeError. In my example I do "Select all" followed by "Send to Library", but the result is also the same with the other options and if nothing is selected. I can't honestly figure out how to use that menu.

**To Reproduce**
Page actions -> Select all
View actions -> Send to Library
Happypanda returns an error on screen and also on the console.

**Logs**
https://pastebin.com/raw/jWMDf8Cy
The seek errors are likely because in the select all popup that shows up no gallery shows thumbnails, so that's where it's failing.

**Environment:**
Linux, Chrome, latest HP version.

**Comment by nonamethanks** [comment]
Seems also selecting a single gallery through its thumbnail menu and then clicking "Send to Library" via the View actions menu returns a TypeError.

```

'An unhandled critical error has occurred '
'An error occurred on client message: b\'{"session": "bef0799ef1804b6a9b83a20d561fa8a3", "name": "webclient", "data": [{"fname": "update_metatags_for_view", "item_type": 1, "metatags": {"inbox": false}, "limit": null, "view_filter": 6, "search_options": {}, "__id__": 104}]}\' '

```

**Comment by twiddli** [comment]
Fixed in `v0.13.0`
